src/field_theory/krull_topology.lean

```diff
@@ -270,3 +273,73 @@ lemma krull_topology_t2 (K L : Type*) [field K] [field L] [algebra K L]
   end }
 
 end krull_t2
+
+section totally_disconnected
+
+/-- Let `X` be a topological space, and suppose that for all distinct `x,y ∈ X`, there
```

Comment:
Please move this to somewhere close to the definition of `is_totally_disconnected`.

Reply:
I have moved it. I'm leaving the conversation unresolved so that you can decide if it's in the right place.